docs/3.0/develop/transactions.mdx

```diff
@@ -63,7 +63,12 @@ def quality_test():
 def pipeline(contents: str):
     with transaction():
         write_file(contents)
+        sleep(2)
```

Comment:
what's the idea with this sleep call?

Reply:
Then you can see the file pop into existence and then see it disappear when the rollback fires. Otherwise I at least couldn't see it get deleted because it happened instantly.

Reply:
ahhhh gotcha -- could you add a comment or something to that effect? `# sleeping to give you a chance to see the file`